src/SFA.DAS.FindEpao.sln

```diff
@@ -5,32 +5,32 @@ VisualStudioVersion = 16.0.30413.136
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.FindEpao.Web", "SFA.DAS.FindEpao.Web\SFA.DAS.FindEpao.Web.csproj", "{A62A4E4D-537D-440A-B00F-42ACE7A3D21F}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.FindEpao.Web.UnitTests", "SFA.DAS.FindEpao.Web.UnitTests\SFA.DAS.FindEpao.Web.UnitTests.csproj", "{0FF48677-BCDA-4A0E-A63B-6E81609B470D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.FindEpao.Web.UnitTests", "SFA.DAS.FindEpao.Web.UnitTests\SFA.DAS.FindEpao.Web.UnitTests.csproj", "{0FF48677-BCDA-4A0E-A63B-6E81609B470D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SFA.DAS.FindEpao.Web.AcceptanceTests", "SFA.DAS.FindEpao.Web.AcceptanceTests\SFA.DAS.FindEpao.Web.AcceptanceTests.csproj", "{E6170F8C-D5C4-4831-8F32-8CA08936A727}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.FindEpao.Web.AcceptanceTests", "SFA.DAS.FindEpao.Web.AcceptanceTests\SFA.DAS.FindEpao.Web.AcceptanceTests.csproj", "{E6170F8C-D5C4-4831-8F32-8CA08936A727}"
```

Comment:
Can we back out this change? shouldn't be any changes to the sln file for the work done.

Reply:
Backed-out